backend/Dockerfile

```diff
@@ -8,10 +8,12 @@ COPY package-lock.json ./
 
 COPY tsconfig.json ./
 
+copy build.ts ./
```

Comment:
I think typo here unless Dockerfile lowercase is okay, if you've tested this its probably ifne